.github/workflows/main.yml

```diff
@@ -135,6 +135,18 @@ jobs:
       run: |
         sh assets/scripts/install.sh
         bin/chezmoi --version
+    - name: test-install.ps1
+      shell: pwsh
+      run: |
+        if (Test-Path -Path bin/chezmoi.exe) { Remove-Item -Force bin/chezmoi.exe }
+        assets/scripts/install.ps1
+        bin/chezmoi --version
+    - name: test-install.ps1-irm
+      shell: pwsh
+      run: |
+        if (Test-Path -Path bin/chezmoi.exe) { Remove-Item -Force bin/chezmoi.exe }
+        iex "&{$(irm 'https://raw.githubusercontent.com/twpayne/chezmoi/${{ env.GITHUB_SHA }}/assets/scripts.install.ps1')}"
```

Comment:
Should this be `/assets/scripts/install.ps1` instead of `/assets/scripts.install.ps1`? (replace `.` with `/`)?

Reply:
Yes